src/ResourceManager/MachineLearningCompute/Commands.MachineLearningCompute/Cmdlets/UpdateAzureRmMlOpClusterSystemService.cs

```diff
@@ -0,0 +1,92 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Commands.MachineLearningCompute.Models;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+using Microsoft.Azure.Management.MachineLearningCompute;
+using Microsoft.Azure.Management.MachineLearningCompute.Models;
+using Microsoft.Rest.Azure;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.MachineLearningCompute.Cmdlets
+{
+
+    [Cmdlet(VerbsData.Update, CmdletSuffix + "SystemService", SupportsShouldProcess = true)]
+    [OutputType(typeof(UpdateSystemServicesResponse))]
```

Comment:
@shutchings same comment about wrapping this in a PS class

Reply:
Done.